src/components/TasksTable/index.jsx

```diff
@@ -76,7 +79,13 @@ export default class TasksTable extends Component {
         <DataTable
           items={data}
           renderRow={item => (
-            <TableRow hover tabIndex={-1} key={`${item.summary}`}>
+            <TableRow
+              hover
+              tabIndex={-1}
+              key={`${item.summary}`}
+              component="a"
+              href={item.url}
+              className={classes.link}>
```

Comment:
When we are dealing with an external link, the convention is navigate the user to the new page via a new window. In code, we do that with `target="_blank"`. Also, to make sure the new window open doesn't have access to the window who opened it (tabnabbing), we add `rel="noopener noreferrer"`. For more information on this, you can refer to this [article](https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/).

```jsx
<TableRow
  target="_blank"
  rel="noopener noreferrer"
  // ...
/>
```